src/common/SPELLS/shadowlands/covenants/priest.js

```diff
@@ -12,6 +12,20 @@ export default {
   //endregion
 
   //region Venthyr
-
+  MIND_GAMES: {
```

Comment:
At least currently, the spell is called Mindgames, no space. Just for clarity for other contributors it should match the format blizzard has.

Reply:
I've been looking at the spell on wowhead all morning and never noticed it was in 1 word! Thanks